src/app/editor/shared/services/record-fixer.service.ts

```diff
@@ -27,46 +27,81 @@ import { JsonUtilService } from './json-util.service'
 
 @Injectable()
 export class RecordFixerService {
-  
+
   private emptyValueService: EmptyValueService = new EmptyValueService();
   private jsonUtilService: JsonUtilService = new JsonUtilService();
 
+  // TODO: return fixed record!
```

Comment:
I don't think I will add it now, unless you think that I should.

It is OK, I just prefer usually to return, that requires extra thing here (copying all record into empty object), that's why I put `TODO` for now, so that I can think about it later.